.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Go
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    outputs:
+      coverage: ${{ steps.coverage.outputs.coverage }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Test
```

Comment:
what do you think about adding make build as a workflow/job?

Reply:
I thought `make test` covered it, but it seems that it doesn't really do that. So, sure, I've added it now